selfdrive/ui/qt/offroad/wifiManager.cc

```diff
@@ -479,8 +488,6 @@ bool WifiManager::tetheringEnabled() {
 
 void WifiManager::changeTetheringPassword(const QString &newPassword) {
   tetheringPassword = newPassword;
-  if (isKnownNetwork(tethering_ssid.toUtf8())) {
-    forgetNetwork(tethering_ssid.toUtf8());
-  }
+  forgetNetwork(tethering_ssid.toUtf8());
```

Comment:
why did we check this before?

Reply:
I added it forgetting that I also added the check in `forgetNetwork` for stuff like this. The previous clear_connections command just iterated through all the connections and performed no action if it didn't exist.